TTHAnalysis/python/tools/higgsDiffCompTTH.py

```diff
@@ -103,10 +116,56 @@ def analyze(self, event):
         pTTrueGen       = getattr(event,'%spTTrueGen'%self.label)
         pTTrueGenPlusNu = getattr(event,'%spTTrueGenPlusNu'%self.label)
         
+        allLeps     = Collection(event,"LepGood","nLepGood")
```

Comment:
I think this block should be removed: the information on the leptons is already produced when running higgsDiffRecoTTH.py: if you need additional variables please add them there.
Here you then access the previously produced diffRecoTTH friend tree.
Do you have any special need that is not accomodated by the variables in higgsDiffGenTTH and higgsDiffRecoTTH?